web/app/style/colors.less

```diff
@@ -24,3 +24,8 @@
 @color-delete: #E44E4E;
 @color-label: #888;
 @color-label-darker: #666;
+
+// My Data colors
+@color-curious-blue: #149ECB;
```

Comment:
Do we really need another blue for My Data (which would be different from the general Oma Opintopolku color scheme)?